legend-engine-xt-nonrelationalStore-mongodb-grammar-integration/src/test/java/org/finos/legend/engine/language/pure/grammar/integration/TestMongoDBAllCompiler.java

```diff
@@ -0,0 +1,144 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.language.pure.grammar.integration;
+
+import org.finos.legend.engine.language.pure.compiler.test.TestCompilationFromGrammar;
+import org.junit.Test;
+
+public class TestMongoDBAllCompiler extends TestCompilationFromGrammar.TestCompilationFromGrammarTestSuite
+{
+
+    @Override
+    protected String getDuplicatedElementTestCode()
```

Comment:
this should test duplicated elements within the ###MongoDB, but I dont see that used here?